src/graphql/account.rs

```diff
@@ -973,13 +976,15 @@ fn sign_in_actions(
     account_map: &Table<types::Account>,
     client_ip: Option<SocketAddr>,
     username: &str,
+    expiry_cutoff: i64,
 ) -> Result<AuthPayload> {
     let (token, expiration_time) =
         create_token(account.username.clone(), account.role.to_string())?;
     account.update_last_signin_time();
     account_map.put(account)?;
 
     insert_token(store, &token, username)?;
+    revoke_expired_tokens(store, expiry_cutoff, username)?;
```

Comment:
What do you think about moving this cleanup logic into a separate, scheduled background task instead of running it during the sign-in request? This would keep the sign-in endpoint consistently fast and prevent potential delays if there are many tokens in the database. It also simplifies error handling. A cleanup failure in a background job wouldn't result in an error in signing in.

Reply:
Processing this cleanup logic through a scheduled background task seems to be a more reasonable approach. Accordingly I’ll exclude this part from the current PR changes.

Reply:
I created https://github.com/aicers/patio/issues/419